code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -65,14 +65,17 @@ There are several things that need to be remembered:
 	return
 
 /mob/living/carbon/human/update_body(redraw = FALSE)
-	dna.species.handle_body(src)
+	if(dna && dna.species)
```

Comment:
why was this needed? it is invalid for a non-qdeleted humantype mob to not have a dna and species.